modules/external-roles/README.md

```diff
@@ -0,0 +1,32 @@
+This module is for creating specific iam roles in specific projects for existing service account.
```

Comment:
`This module is intended to create specific IAM roles in specific projects for the existing service account.`